docs/sources/administration/data-source-management/teamlbac/create-teamlbac-rules/index.md

```diff
@@ -123,3 +123,14 @@ A user that is part of Team A will have access to logs that match `namespace="de
 A user that is part of Team A and part of Team B will have access to logs that match `namespace="dev"`.
 
 A user that is not part of Team A and part of Team B, that is `Editor` or `Viewer` will have access to all logs (due to the query permission for the user).
+
+### Task 6: User A is Admin and part of Team B
+
+We have team B, user A is part of Team B and has an `Admin`.
```

Comment:
Nit: maybe specify that you're talking about org admin and not a team admin here?
```suggestion
We have team B, user A is part of Team B and has an `Admin` basic role.
```